internal/postprocessor/main.go

```diff
@@ -311,26 +312,61 @@ func (p *postProcessor) getDirs() []string {
 
 func (p *postProcessor) MoveSnippets() error {
 	log.Println("moving snippets")
-	dirs := p.getDirs()
-	for _, dir := range dirs {
+	for _, clientRelPath := range p.config.ClientRelPaths {
+		// OwlBot dest relative paths in ClientRelPaths begin with /, so the
+		// first path segment is the second element.
+		moduleName := strings.Split(clientRelPath, "/")[1]
+		if len(p.modules) > 0 && !contains(p.modules, moduleName) {
+			continue
+		}
+		snpDir := filepath.Join(p.googleCloudDir, clientRelPath, "internal", "snippets")
+		if _, err := os.Stat(snpDir); err != nil {
+			continue
+		}
 
-		snpDirs, err := filepath.Glob(filepath.Join(dir, "apiv*", "internal"))
+		toDir := filepath.Join(p.googleCloudDir, "internal", "generated", "snippets", clientRelPath)
+		log.Printf("deleting old snippets and metadata at %s", toDir)
+		err := os.RemoveAll(toDir)
```

Comment:
nit: combine this the the next line: `if err := os.RemoveAll(toDir); err != {`. Same below as well

Reply:
Fixed, thanks!